Gopkg.toml

```diff
@@ -36,7 +37,14 @@ version="v1.4.7"
   name = "github.com/intel/multus-cni"
   branch = "master"
 
+[[override]]
+  name = "kubevirt.io/kubevirt"
+  version = "=v0.20.4"
+
 [[constraint]]
-  name= "kubevirt.io/kubevirt"
-  source = "https://github.com/kubevirt/kubevirt.git"
-  version = "v0.17.0"
+  name = "kubevirt.io/client-go"
+  version = "=v0.20.4"
+
+[[override]]
```

Comment:
why do we need CDI?

Reply:
This is an hard request by the kubevirt project and the version should be override (it takes the master) this is way is override and not in the request list of the project